### PR TITLE
chore(md): add some testing tips for development

### PR DESCRIPTION
**Pull request by zarov**
Added some commands I'm using for development, I find it useful, maybe it can interest someone ?

### CODE REVIEW

File: CODING.md [hunk 1]
```diff
@@ -64,6 +64,20 @@ Note: Chrome in headless mode doesn't support the WebGL `EXT_frag_depth`
...
+### Useful commands for continuous testing
+
+If you wish to have unit tests to continuously run, you can append to your
+running command `-- --watch`. You will be getting in this case `npm run
+test-unit -- --watch`.
+
+If you want to work on a single test and debug it, without having all the extra
+output, you can use this command:
+```sh
+./node_modules/.bin/cross-env BABEL_DISABLE_CACHE=1 \
```
Comment (tbroyer) [inline CODING.md]: How about `npx cross-env` rather than `./node_modules/.bin/cross-env` (and similarly for `mocha`)

File: CODING.md [hunk 2]
```diff
@@ -64,6 +64,20 @@ Note: Chrome in headless mode doesn't support the WebGL `EXT_frag_depth`
...
+  ./node_modules/.bin/mocha --require @babel/register \
+  test/unit/3dtiles.js -- --watch # replace 3dtiles.js with the correct filename
```
Comment (tbroyer) [inline CODING.md]: Would it then be a good idea to add a `base-test-unit` NPM script that would call mocha without any "target", and rewrite `test-unit` as: `npm run base-test-unit -- test/unit`? then this command could become `npm run base-test-unit -- --watch test/unit/3dtiles.js`.

File: CODING.md (hunk 1 above)
Comment (zarov) [reply CODING.md]: Oh I finally understand what `npx` is, thanks :smile: Will do

File: CODING.md (hunk 2 above)
Comment (zarov) [reply CODING.md]: Yeah why not, this seems like a good idea

**Comment by gchoqueux** [comment]
This Pr could be merged?

**Comment by zarov** [comment]
I need to get back to it first, I think I wanted to change things.

**Comment by zarov** [comment]
@gchoqueux this is good to merge

**Review by gchoqueux — APPROVED** [review]

**Pull request merged**